scripts/ci-scripts/start-server.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
We need to discuss how we are doing this, now that we are all moving to M1 macs.  In my opinion, we should start testing against the QA server.

Reply:
It doesn't seem that we have an option here as long as BAAS doesn't come in an M1 variant.
When it does, though, it would be nice to have the option to run locally IMHO.